spec/scss_converter_spec.rb

```diff
@@ -112,8 +121,13 @@ def converter(overrides = {})
         expect(verter.sass_configs[:style]).to eql(:compressed)
       end
 
-      it "defaults style to :compact" do
-        expect(verter.sass_configs[:style]).to eql(:compact)
+      it "defaults style to :expanded for sass-embedded or :compact for sassc" do
+        case sass_implementation
+        when "sass-embedded"
```

Comment:
Lets make more use of the `GlobalSharedContext` module to inline these expressions.

```ruby
module GlobalSharedContext
  extend RSpec::SharedContext

  let(:sass_implementation) { ENV["SASS_IMPLEMENTATION"] }
  let(:sass_embedded?) { sass_implementation == "sass-embedded" }
end
```
```ruby
it "does something" do
  o = sass_embedded? ? a : b
  expect(converter.something).to eql(o)
end
```

Sure `case` is more scalable, but unnecessary for the near future.